src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -44,7 +50,25 @@ limitations under the License.
       <arg name="data" type="CHAR_STRING"/>
     </command>
 
+    <command source="server" code="0x01" name="StopAppResponse" optional="false">
```

Comment:
There is no StopAppResponse in the spec.  The response is "Launcher".  Why are we adding these non-spec bits?

Reply:
Yes, I'm not sure if it's still the case, but for every corresponding command to I've had to create a patter so that response command works. The pattern is:

Name of the response command has to be the same as the request command + the string response. So for StopApp, it has to be StopAppResponse, otherwise it wouldn't generate the code.

Code of the command has to be the same as the request command. In this case StopApp has the code 0x01 and the response command also has 0x01.